dockerfiles/che/rhel.Dockerfile

```diff
@@ -34,10 +32,10 @@ RUN mkdir /logs /data && \
     chown -R user /home/user && \
     chmod -R g+rwX /home/user && \
     find /home/user -type d -exec chmod 777 {} \; && \
+    chmod 0777 /home/user/cacerts && \
```

Comment:
why do we need this kind of permissions on this file ? it looks like a big hole

Reply:
I figured doing this would allow the later change to 644.